nqueens.c

```diff
@@ -94,14 +97,14 @@ int main(int argc, char *argv[]) {
   a = (char *) alloca (n * sizeof (char));
   res = 0;
 
-  struct timeval t1, t2;
-  gettimeofday(&t1,0);
+  ctimer_t t;
+  ctimer_start(&t);
 
   res = nqueens(n, 0, a);
 
-  gettimeofday(&t2,0);
-  unsigned long long runtime_ms = (todval(&t2)-todval(&t1))/1000;
-  printf("%f\n", runtime_ms/1000.0);
+  ctimer_stop(&t);
+  ctimer_measure(&t);
+  ctimer_print(t);
```

Comment:
I'm getting a compile error due to this line:
`error: too few arguments to function call, expected 2, have 1`
```suggestion
  ctimer_print(t, "nqueens");
```